wandb/sdk/launch/runner/local_container.py

```diff
@@ -9,7 +9,7 @@
 import wandb
 from wandb.sdk.launch.builder.abstract import AbstractBuilder
 
-from .._project_spec import LaunchProject, get_entry_point_command
```

Comment:
Does `get_entry_point_command` still exist?

Reply:
Yeah, its used in other runners.